config.yml

```diff
@@ -6,9 +6,8 @@
 ## Updated: 12/07/2023
 
 ---
-cluster_name: "CW23"
-location: "EH8"
-api: carbonintensity.org.uk
+location: "EH8 9BT"
```

Comment:
Any reasons why the model `config` file has a full postcode instead of just the first part? This may give the impression to users that this is expected, or that the API uses the whole postcode, while the results are actually only for EH8 here (our code is able to extract the first part, but maybe best to encourage users to only provide it anyway).

Reply:
I think it was an example postcode that got used early in the development before we realised we didn't need to specify a full postcode. Go ahead and change it to a shorter postcode.

Reply:
Just did.